cmd/registry/metrics/metrics.go

```diff
@@ -0,0 +1,51 @@
+package metrics
+
+import (
+	"github.com/apigee/registry/rpc"
+)
+
+// ComputeMetrics will compute the metrics proto for a list of Classified Diffs.
+func ComputeMetrics(calculatedDiffs []*rpc.ClassifiedChanges) *rpc.Metrics {
+	stats := calculateStats(calculatedDiffs)
+	return caluclateMetrics(stats)
+}
+
+func calculateStats(calculatedDiffs []*rpc.ClassifiedChanges) *rpc.Stats {
+	totalBreakingChanges := 0
+	totalNonbreakingChanges := 0
```

Comment:
Nit: If you declare these as int64 you won't need to cast them later. It doesn't matter much in this case, but if these sums ever exceeded 32 bits then declaring them as int64 would be really important.

```
var (
	totalBreakingChanges int64 = 0
	totalNonbreakingChanges int64 = 0
)
```

Reply:
Done